### PR TITLE
Fix E2E issues in CI

### PR DESCRIPTION
**Pull request by xmano**
Run tests for all apps
Use chrome browser as default for test instead of electron

### CODE REVIEW

**Comment by xmano** [comment]
Need to update Docker Image to include latest chrome browser:

```
# The image shall be pushed to dockerhub:
#    docker build -t blockframes/ci:latest .
#    docker push blockframes/ci:latest
```

I have to understand this step and will continue fixing the issue after this.

**Comment by xmano** [comment]
I don't have permission to push Docker image to blockframes/ci
Let me know how to resolve this.

I think we can go ahead with the config of E2E with chrome browser.

File: angular.json [hunk 1]
```diff
@@ -12,13 +12,8 @@
         "lint": {
```
Comment (phpgeekfr) [inline angular.json:12]: can you revert the "format" changes ? so we only see what was updated?

File: .circleci/config.yml [hunk 2]
```diff
@@ -140,18 +140,18 @@ commands:
...
+          name: Uploading screenshots
+          path: ~/repo/dist/cypress/apps
 orbs:
   do-exclusively: miyajan/do-exclusively@0.1.0
   slack: circleci/slack@3.4.0
 executors:
   blockframes-ci:
     docker:
-      - image: blockframes/ci:latest
+      - image: zenmano/ci:browsers-latest
```
Comment (phpgeekfr) [inline .circleci/config.yml:154]: I think we should stick with blockframes image. Is this for test purposes?

File: apps/financiers/financiers-e2e/src/integration/app.spec.ts [hunk 3]
```diff
@@ -1,13 +1,5 @@
...
   beforeEach(() => cy.visit('/'));
 
-  it('should display welcome message', () => {
-    // Custom command example, see `../support/commands.ts` file
-    cy.login('my-email@something.com', 'myPassword');
-
-    // Function helper example, see `../support/app.po.ts` file
-    getGreeting().contains('Welcome to financiers!');
-  });
+  it.skip('should display welcome message', () => {});
```
Comment (phpgeekfr) [inline apps/financiers/financiers-e2e/src/integration/app.spec.ts:4]: you skipped it because it crash ?

File: angular.json (hunk 1 above)
Comment (xmano) [reply angular.json:12]: Ok will do it

File: .circleci/config.yml (hunk 2 above)
Comment (xmano) [reply .circleci/config.yml:154]: Yes i need a way to publish to blockframes. Maybe @george43g can help here?

File: apps/financiers/financiers-e2e/src/integration/app.spec.ts (hunk 3 above)
Comment (xmano) [reply apps/financiers/financiers-e2e/src/integration/app.spec.ts:4]: Tests are not ready.. We need to add this later. Not in this PR scope.

File: .circleci/config.yml (hunk 2 above)
Comment (phpgeekfr) [reply .circleci/config.yml:154]: yes, ask @george43g for this.

File: angular.json [hunk 4]
```diff
@@ -12,13 +12,8 @@
...
           "builder": "@angular-devkit/build-angular:tslint",
           "options": {
-            "tsConfig": [
```
Comment (george43g) [inline angular.json:15]: definitely revert all of these formatting changes

File: .circleci/config.yml [hunk 5]
```diff
@@ -152,7 +152,7 @@ orbs:
 executors:
   blockframes-ci:
     docker:
-      - image: blockframes/ci:latest
```
Comment (george43g) [inline .circleci/config.yml:155]: revert these changes as well

File: .circleci/Dockerfile [hunk 6]
```diff
@@ -6,7 +6,7 @@
 #    docker build -t blockframes/ci:latest .
 #    docker push blockframes/ci:latest
 #
-FROM circleci/node:12
+FROM circleci/node:12-browsers
```
Comment (george43g) [inline .circleci/Dockerfile:9]: and revert these as well. I will add it in and push and merge it in now

**Review by george43g — CHANGES_REQUESTED** [review]

File: .circleci/Dockerfile (hunk 6 above)
Comment (xmano) [reply .circleci/Dockerfile:9]: ok

File: .circleci/config.yml (hunk 5 above)
Comment (xmano) [reply .circleci/config.yml:155]: ok

File: .circleci/config.yml (hunk 2 above)
Comment (xmano) [reply .circleci/config.yml]: ok

**Review by george43g — APPROVED** [review]

**Comment by phpgeekfr** [comment]
Finally, thanks :)

File: apps/financiers/financiers-e2e/tsconfig.json [hunk 7]
```diff
@@ -1,7 +1,10 @@
-{
+﻿{
   "extends": "../../../tsconfig.base.json",
-  "files": [],
+  "compilerOptions": {
+    "types": ["cypress", "node", "cypress-wait-until"],
```
Comment (phpgeekfr) [inline apps/financiers/financiers-e2e/tsconfig.json:4]: extra comma here

Comment (xmano) [reply apps/financiers/financiers-e2e/tsconfig.json:4]: removed

**Pull request merged**